app/controllers/profiles_controller.rb

```diff
@@ -67,6 +67,10 @@ def set_profile
   # Only allow a list of trusted parameters through.
   def profile_params
     params.fetch(:profile, {})
-    # params.require(:profile).permit(:description, :avatar_url, :other_attribute)
+    {
```

Comment:
Just a note to follow-up at some point to implement `params.require` and `.permit` since they are important for security. We'll have to update this code either way when it's time to set a real value for the `description` field. For now, this is good just to get it working.